convention/src/main/java/com/appcoins/wallet/convention/Config.kt

```diff
@@ -5,8 +5,8 @@ import org.gradle.api.JavaVersion
 object Config {
   val android = AndroidConfig(
     minSdk = 21,
-    targetSdk = 33,
-    compileSdkVersion = 33,
+    targetSdk = 34,
+    compileSdkVersion = 34,
     buildToolsVersion = "30.0.3",
```

Comment:
this compileSdkVersion recommend to use a newer version of build gradle, can you see the impact of this or update the gradle?

Reply:
I had no impact in that regard.